conf/kamailio/kamailio.cfg

```diff
@@ -320,10 +320,10 @@ modparam("sanity", "autodrop", 0)
 # ----- tm params -----
 # auto-discard branches from previous serial forking leg
 modparam("tm", "failure_reply_mode", 3)
-# default retransmission timeout: 30sec
-modparam("tm", "fr_timer", 30000)
-# default invite retransmission timeout after 1xx: 120sec
-modparam("tm", "fr_inv_timer", 120000)
+# default retransmission timeout: 5sec
+modparam("tm", "fr_timer", 5000)
+# default invite retransmission timeout after 1xx: 10sec
+modparam("tm", "fr_inv_timer", 10000)
```

Comment:
Why have you reduced these values?

Reply:
Thought It will help speeding up the failover of the carriers. Any suggestions?

Reply:
Sounds good (as we use UDP), let's keep the reduced timeouts.